.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,31 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: pull-request
+    attributes:
+      label: Pull Request
+occurred
```

Comment:
@MicahZoltu I think you accidentally committed to this PR

Reply:
It was on purpose, I was trying to fix the spelling error that codespell gave.  I have no idea how I failed so badly here.  😖  Tried again, I think I got it right this time.  😬